src/pricecypher/models/datasets.py

```diff
@@ -0,0 +1,99 @@
+from dataclasses import field
+from datetime import datetime
+from typing import Optional, List, Union
+from marshmallow_dataclass import dataclass
+
+from pricecypher.models.namespaced_schema import NamespacedSchema
+
+
+@dataclass(base_schema=NamespacedSchema, frozen=True)
+class Scope:
+    id: int = field(compare=True)
+    data_version_id: int = field(compare=False)
+    order: Optional[int] = field(compare=False)
+    type: str = field(compare=False)  # TODO enum
+    representation: Optional[str] = field(compare=False)  # TODO enum
+    name_dataset: str = field(compare=False)
+    name_human: Optional[str] = field(compare=False)
```

Comment:
```suggestion
    name_human: str = field(compare=False)
```
I don't think the `name_human` is nullable, at least not currently.

Reply:
I just checked on some deployed versions, running on the newest version of dataset service, and the name_human field is nullable.

Reply:
Oh indeed! For some reason in my local DB it is not, but looking at the migrations it should be nullable, indeed.